fe/fe-core/src/main/java/org/apache/doris/analysis/Analyzer.java

```diff
@@ -311,7 +310,6 @@ public GlobalState(Catalog catalog, ConnectContext context) {
             rules.add(RewriteDateLiteralRule.INSTANCE);
             rules.add(RewriteEncryptKeyRule.INSTANCE);
             rules.add(RewriteAliasFunctionRule.INSTANCE);
-            rules.add(RewriteLikePredicateRule.INSTANCE);
```

Comment:
Is it ok for 'non-vec' without any be changed ?

Reply:
> Is it ok for 'non-vec' without any be changed ?

I searched and deleted all code whitch use  `RewriteLikePredicateRule`.

Reply:
I understand that the reason why FE added this rule before is PR because the direct execution of non-vectorized code will make mistakes.
If only changing FE but not BE, will the non-vectorized code now go wrong?